### ISSUE TITLE
FZF_COMPLETION_TRIGGER doesn't use FZF_DEFAULT_COMMAND as input

### ISSUE THREAD

**Issue by thomcom** [junegunn/fzf#406]
I've updated my `FZF_DEFAULT_COMMAND='ag -g ""'` in order to respect `.gitignore`.  It works fine using `CTRL-T`, but if I use `**<TAB>` completion, I still see `node_modules`.  How do I use `ag -g ""` with `FZF_COMPLETION_TRIGGER`?

**Comment by junegunn** [comment]
Unfortunately it's not configurable at the moment. `ag -g ""` is not an ideal command since it does not list the names of directories. If you see [the source code of fuzzy completion](https://github.com/junegunn/fzf/tree/master/shell), you'll notice that the command used is context-dependent (e.g. `rmdir **` will only give the list of directories). And I haven't decided how to make them configurable. Currently you have to copy and edit the code directly.

**Comment by thomcom** [comment]
Ok, thanks!  That explained it clearly enough that was easy to add locally.  I see why you haven't decided how to configure it yet!  :)

**Comment by junegunn** [comment]
Alright, I'll close this for now.

**Comment by jasonkuhrt** [comment]
I just encountered this issue too. Would be nice to have a solution. Using `fzf` to navigate from `/projects` is nifty but unusable without ignoring git ignored files. Maybe this should be opened and closed only once an interface for configuration has been designed?

**Comment by junegunn** [comment]
@jasonkuhrt This is an old issue. See https://github.com/junegunn/fzf#settings
